packages/st2/rpm/st2.spec

```diff
@@ -3,7 +3,7 @@
 %define svc_user st2
 %define stanley_user stanley
 %define packs_group st2packs
-#define epoch %(_epoch=`echo $ST2PKG_VERSION | grep -q dev || echo 1`; echo "${_epoch:-0}")
+%define epoch %(_epoch=`echo $ST2PKG_VERSION | grep -q dev || echo 1`; echo "${_epoch:-0}")
```

Comment:
Please keep the line commented. I think we don't use `epoch`

Reply:
@armab  Can we remove it then? It was causing a warning everytime that as annoying?

Reply:
Sure, I'm good with that :+1: